documentation/book/faq.adoc

```diff
@@ -1,9 +1,52 @@
+
+After installation, the Cluster Operator will run as a regular `Deployment`; any non-admin user with privileges to access the `Deployment` can configure it.
+
+[id='normal-user-access-custom-resources-{context}']
+By default, normal users will not have the privileges necessary to manipulate the custom resources, such as `Kafka`, `KafkaConnect` and so on, which the Cluster Operator deals with.
+These privileges can be granted using normal RBAC resources by the cluster administrator. See xref:assembly-getting-started-strimzi-admin-str[this procedure] for more details of how to do this.
+
+
+=== Why does the Cluster Operator require the ability to create `ClusterRoleBindings`? Is that not a security risk?
+
+{PlatformProductName} has built-in link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping[privilege escalation prevention^]. 
+That means that the Cluster Operator cannot grant privileges it does not have itself. 
+Which in turn means that the Cluster Operator needs to have the privileges necessary for _all_ the components it orchestrates.
+
+In the context of this question there are two places where the Cluster Operator needs to create bindings to `ClusterRoleBindings` to `ServiceAccounts`:
+
+. The Topic Operator and User Operator need to be able to manipulate `KafkaTopics` and `KafkaUsers`, respectively.
+The Cluster Operator therefore needs to be able to grant them this access, which it does by creating a `Role` and `RoleBinding`.
+For this reason the Cluster Operator itself needs to be able to create `Roles` and `RoleBindings` in the namespace that those operators will run in.
+However, because of the privilege escalation prevention, the Cluster Operator cannot grant privileges it does not have itself (in particular it cannot grant such privileges in namespace it cannot access).
+
+. When using rack aware partition assignment, {ProductName} needs to be able to discover the failure domain (for example, the Availability Zone in AWS) of the node on which a broker pod is assigned.
+In order to do this the broker pod needs to be able to get information about the `Node` it is running on.
+A `Node` is a cluster-scoped resource, so access to it can only be granted via a `ClusterRoleBinding` (not a namespace-scoped `RoleBinding`). 
+Therefore the Cluster Operator need to be able to create `ClusterRoleBindings`.
+But again, because of the privilege escalation prevention, the Cluster Operator cannot grant privileges it does not have itself (so it cannot, for example, create a `ClusterRoleBinding` to a `ClusterRole` which the Cluster Operator is not already bound to via some other `ClusterRoleBinding`).
```

Comment:
because of privilege escalation prevention

to which the Cluster Operator is not already bound

Reply:
> to which the Cluster Operator is not already bound

Not sure what you intended that to replace.

Reply:
Intended to replace "which the Cluster Operator is not already bound to". Thanks!

Reply:
That's not quite the same meaning: It doesn't have to be bound to the _same_ `ClusterRole`, it just needs to have the same set of permissions as the one it's trying to bind to. It happens that the easiest to ensure that is to use the same `ClusterRole`, but it doesn't have to work like that.

Reply:
Happy for you to reword as appropriate.

Reply:
Please use "that" instead of "which" as per the style guide.

Reply:
Better?

> (so it cannot, for example, create a `ClusterRoleBinding` to a `ClusterRole` to grant privileges that the Cluster Operator does not not already have)